Shelfie.Identity/Shelfie.Identity.BusinessLogic/UseCases/LoginUser/LoginUserCommandHandler.cs

```diff
@@ -33,6 +38,12 @@ public LoginUserCommandHandler(UserManager<IdentityUser> userManager, IJwtServic
 
         var token = _jwtService.GetToken(authClaims);
 
-        return new JwtSecurityTokenHandler().WriteToken(token);
+        result = new OperationResult<string?>
```

Comment:
f.e. 
return OperationResult.Succes(new JwtSecurityTokenHandler().WriteToken(token)).

value for indicating if succesfull should be encapsulated